src/main/java/org/openjdk/shenandoah/ShenandoahVisualizer.java

```diff
@@ -460,23 +460,23 @@ public synchronized void renderLegend(Graphics g) { //delete the static
 
             summary_numbers.put("Trash", new Integer(snapshot.trashCounter()));
 
-            summary_numbers.put("Fully Live, 100% TLAB Allocs", new Integer(snapshot.tlabCounter()));
+            summary_numbers.put("TLAB Allocs", new Integer(snapshot.tlabCounter()));
```

Comment:
With the "Fully Live" and "1/3 Live" stuff gone, this makes much more sense.

Reply:
Yes, I delete it a bit too late.